packages/flutter_tools/lib/src/runner/flutter_command_runner.dart

```diff
@@ -146,13 +145,20 @@ class FlutterCommandRunner extends CommandRunner<void> {
 
   @override
   String get usageFooter {
-    return wrapText('Run "flutter help -v" for verbose help output, including less commonly used options.');
+    return wrapText('Run "flutter help -v" for verbose help output, including less commonly used options.',
+      columnWidth: outputPreferences.wrapColumn,
```

Comment:
Should `wrapText` take a single named arg for the output preferences instead?

Reply:
There are a few properties that don't come from output preferences. I think the API needs some refactoring, but its used quite a bit so I'd rather do that in a different PR.